internal/cmd/beta/network-interface/create/create.go

```diff
@@ -0,0 +1,247 @@
+package create
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"regexp"
+
+	"github.com/goccy/go-yaml"
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/print"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/iaas/client"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+	"github.com/stackitcloud/stackit-sdk-go/services/iaas"
+)
+
+const (
+	networkIdFlag        = "network-id"
+	allowedAddressesFlag = "allowed-addresses"
+	ipv4Flag             = "ipv4"
+	ipv6Flag             = "ipv6"
+	labelFlag            = "labels"
+	nameFlag             = "name"
+	securityGroupsFlag   = "security-groups"
+	nicSecurityFlag      = "nic-security"
+
+	nameRegex           = `^[A-Za-z0-9]+((-|_|\s|\.)[A-Za-z0-9]+)*$`
+	securityGroupsRegex = `^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$`
+	defaultEnabled      = true
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	NetworkId        *string
+	AllowedAddresses *[]iaas.AllowedAddressesInner
+	Ipv4             *string
+	Ipv6             *string
+	Labels           *map[string]string
+	Name             *string // <= 63 characters + regex  ^[A-Za-z0-9]+((-|_|\s|\.)[A-Za-z0-9]+)*$
+	NicSecurity      *bool
+	SecurityGroups   *[]string // = 36 characters + regex ^[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$
+}
+
+func NewCmd(p *print.Printer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Creates a network interface",
+		Long:  "Creates a network interface.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`Create a network interface with allowed addresses "1.1.1.1,8.8.8.8,9.9.9.9" with ipv4 address "1.2.3.4" with label "key=value" with name "NAME" with security groups "UUID1,UUID2" with nic security enabled for network with ID "xxx"`,
+				`$ stackit beta network-interface create --network-id xxx --allowed-addresses "1.1.1.1,8.8.8.8,9.9.9.9" --ipv4 1.2.3.4 --labels key=value --name NAME --security-groups "UUID1,UUID2" --nic-security"`,
```

Comment:
```suggestion
				`$ stackit beta network-interface create --network-id xxx --allowed-addresses "1.1.1.1,8.8.8.8,9.9.9.9" --ipv4 1.2.3.4 --labels key=value,key2=value2 --name NAME --security-groups "UUID1,UUID2" --nic-security"`,
```

Reply:
I think it's useful to show what the expected format is when providing multiple labels